src/test/java/com/exasol/AbstractLuaVirtualSchemaIT.java

```diff
@@ -164,4 +165,14 @@ protected void assertRlsQueryThrowsExceptionWithMessageContaining(final String s
         final SQLException exception = assertThrows(SQLException.class, () -> executeRlsQueryWithUser(sql, user));
         assertThat(exception.getMessage(), containsString(expectedMessageFragment));
     }
+
+    protected void assertPushDown(final String sql, final User user, final Matcher<String> matcher) {
+        try (final ResultSet result = executeRlsQueryWithUser("EXPLAIN VIRTUAL " + sql, user)) {
+            result.next();
+            final String pushDownSql = result.getString("PUSHDOWN_SQL");
+            assertThat(pushDownSql, matcher);
+        } catch (final SQLException exception) {
+            throw new AssertionError("Unable to run push-down assertion query:" + exception.getMessage());
```

Comment:
```suggestion
            throw new AssertionError("Unable to run push-down assertion query:" + exception.getMessage(), exception);
```

add cause to get stack trace